Budget Manager/task/src/budget/PurchaseType.java

```diff
@@ -13,6 +13,36 @@
         this.value = value;
     }
 
+    public static String toString(String value) {
```

Comment:
Deze is wat lelijk opgelost, Je kan aan enums meerdere velden meegeven. Dat is hier de nettere oplossing! Voeg een veld toe aan de enum `FOOD("1", "Food")` en de toString method geeft dat veld terug.